sidebars.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
           type: 'category',
           label: 'Experiments 🧪',
           items: [
-            'experiments/overview',
```

Comment:
We probably also want an introduction (not overview) section explaining what experiments mean.
It basically means new features we are trying that we think are useful, but that may disappear.
We make no promise on experimental features.
cc @ievans

Reply:
In particular, some experiments might be promoted to proprietary features that only logged-in users can access.

Reply:
cc @dlukeomalley

Reply:
and we can also say some experiments may also just disappear, because they are a maintenance burden we can't support.
And some might be promoted to regular features.
We could even give some examples of past experiments who got promoted (I think autofix and generic matching should be for example).

Reply:
@aryx we should not remove docs for unsupported experiments, but move them to the deprecated experiments document (so everyone can see that this thing no longer works) in my opinion.

What do you think about an introductory file with the following content:

# Introduction

The experiments category documents experimental features and the way you can use them. As these features are experimental, some of them can be deprecated, others can become generally available (GA), meaning that GA features are fully supported parts of Semgrep.
If a feature is deprecated, its documentation is moved to (TODO link) the Deprecated experiments document. If a feature becomes GA, their docs are moved to a its relevant category outside of the experiments section.

If you see any issues with the experimental features, please [file bugs](https://github.com/returntocorp/semgrep/issues/new/choose), tweak the code, and most importantly share your thoughts!

Reply:
Let me know @dlukeomalley and @aryx what you think about it.

Reply:
looks good to me.